code/modules/antagonists/ert/ert.dm

```diff
@@ -34,12 +34,12 @@
 /datum/antagonist/ert/get_team()
 	return ert_team
 
-/datum/antagonist/ert/New()
-	. = ..()
-	name_source = GLOB.last_names
-
 /datum/antagonist/ert/proc/update_name()
-	owner.current.fully_replace_character_name(owner.current.real_name,"[role] [pick(name_source)]")
+	var/name
```

Comment:
The way I would do this is first call pick(GLOB.last_name) and then check if the client uses a custom name pref, and then assign that pref to name

Reply:
![image](https://user-images.githubusercontent.com/85033680/193482049-e38c113e-2d8f-426d-91fc-285be48d97f2.png)
it works fine